SpatialGDK/Extras/schema/relevant.schema

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
the schema should probably be named AlwaysRelevant.schema?

Reply:
I was thinking we could put other Unreal relevant components in here - OwnerOnly etc.

Reply:
I like that plan.